packages/x-date-pickers-pro/src/tests/describeRangeValidation/describeRangeValidation.tsx

```diff
@@ -1,33 +1,37 @@
 /* eslint-env mocha */
 import * as React from 'react';
+import createDescribe from '@mui/monorepo/test/utils/createDescribe';
 import { testDayViewRangeValidation } from './testDayViewRangeValidation';
 import { testTextFieldRangeValidation } from './testTextFieldRangeValidation';
 import { DescribeRangeValidationInputOptions } from './describeRangeValidation.types';
 
 const TEST_SUITES = [testDayViewRangeValidation, testTextFieldRangeValidation];
 
-/**
- * Tests various aspects of the range picker validation.
- */
-export function describeRangeValidation(
+function innerDescribeRangeValidation(
   ElementToTest: React.ElementType,
   getOptions: () => DescribeRangeValidationInputOptions,
 ) {
-  describe('Range pickers validation API', () => {
-    const { after: runAfterHook = () => {}, views } = getOptions();
+  const { after: runAfterHook = () => {}, views } = getOptions();
 
-    after(runAfterHook);
+  after(runAfterHook);
 
-    function getTestOptions() {
-      return {
-        ...getOptions(),
-        withDate: views.includes('year') || views.includes('month') || views.includes('day'),
-        withTime: views.includes('hours') || views.includes('minutes') || views.includes('seconds'),
-      };
-    }
+  function getTestOptions() {
+    return {
+      ...getOptions(),
+      withDate: views.includes('year') || views.includes('month') || views.includes('day'),
+      withTime: views.includes('hours') || views.includes('minutes') || views.includes('seconds'),
+    };
+  }
 
-    TEST_SUITES.forEach((testSuite) => {
-      testSuite(ElementToTest, getTestOptions);
-    });
+  TEST_SUITES.forEach((testSuite) => {
+    testSuite(ElementToTest, getTestOptions);
   });
 }
+
+/**
+ * Tests various aspects of the range picker validation.
+ */
+export const describeRangeValidation = createDescribe(
+  'Validation API',
```

Comment:
Maybe we should have kept the original/unique `describe` names? 🤔 
```suggestion
  'Range pickers validation API',
```